airbrakes/hardware/servo.py

```diff
@@ -10,12 +10,21 @@
 
 class Servo:
     """
-    A custom class that represents a servo motor, controlling the extension of the airbrakes.
+    A custom class that represents a servo motor, controlling the extension of the airbrakes. The servo is controlled
+    using the gpiozero library, which provides a simple interface for controlling GPIO pins on the Raspberry Pi.
```

Comment:
don't you want to mention why we're using a bang bang controller and not PID?

Reply:
the servo won't have the bang bang controller, that logic will most likely be in state.update()

Reply:
hm interesting